src/color-picker/color-picker.js

```diff
@@ -50,7 +50,7 @@ const Self = class ColorPicker extends NudeElement {
 			// From sliders
 			let coords = [...this._el.sliders.children].map(el => el.value);
 			this.color = new Color(this.space, coords);
-			this._el.swatch.value = this.color;
+			this._el.swatch.color = this.color;
```

Comment:
Can you just commit this directly since it's an obvious fix and we don't want the PR to hold it up?

Reply:
Sure!